aait/web/group-3/starter_project_group3/components/blog/Pagination.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+interface PaginationProps {
+  currentPage: number;
+  totalPages: number;
+  onPageChange: (page: number) => void;
+}
+
+type Props = PaginationProps & React.HTMLProps<HTMLDivElement>;
+
+const Pagination: React.FC<Props> = ({
+  currentPage,
+  totalPages,
+  onPageChange,
+  className, // This will be used to apply additional class names
```

Comment:
is this comment needed? can't the code be made self-explanatory.

Reply:
Fixed

Reply:
Put inside data/blog/blog.ts file. Will be replaced with real data